packages/odyssey-react-mui/src/CheckboxGroup.tsx

```diff
@@ -53,7 +49,6 @@ const CheckboxGroup = ({
   children,
   errorMessage,
   hint,
-  id: idOverride,
```

Comment:
Where does the `id` on line `63` go?

Reply:
Good catch, I removed it because it is completely unnecessary here.